src/err/mod.rs

```diff
@@ -138,10 +138,6 @@ impl PyErr {
     where
         A: PyErrArguments + Send + Sync + 'static,
     {
-        if unsafe { ffi::PyExceptionClass_Check(ty.as_ptr()) } == 0 {
```

Comment:
The doc comment still seems to refer to normalization even though this does not happen any more? (That sentence is also really hard to read as it is.)

Reply:
The normalisation is still done later (where I moved this check to).

However it's an outdated term which is deprecated in python 3.12, so I'll update the doc 👍